src/components/card/card.jsx

```diff
@@ -5,18 +5,28 @@ import _ from "lodash";
 import { Link } from "gatsby";
 
 import { Category } from "../category";
+import CategoriesYAMLDATA from "../../../content/categories.yaml";
+import { CategorySerializer } from '../../serializer';
 
-import * as cardStyles from "./card.module.scss";
+import * as styles from "./card.module.scss";
 
-const Card = ({ data }) => (
-  <article key={data.id} className={cardStyles.container}>
-    <img className={cardStyles.styledImage} src={data.frontmatter.thumbnail}/>
-    <Category data={data.frontmatter.category}/>
-    <Link className={cardStyles.styledLink} to={`/${_.kebabCase(data.frontmatter.permalink)}`}>
-      {data.frontmatter.title}
+const Card = ({ data }) => {
+
+  const category = CategoriesYAMLDATA.find(({ category }) => (category === data.frontmatter.category));
+  const categoryBlog = CategorySerializer.deSerialize(category);
+
+  return (
+    <Link className={styles.styledLink} to={`/${_.kebabCase(data.frontmatter.permalink)}`}>
+      <article key={data.id} className={styles.container}>
+        <img className={styles.styledImage} src={data.frontmatter.thumbnail}/>
+        <Category data={categoryBlog.displayName}/>
```

Comment:
Does not the data prop has this value?

Reply:
No, I search and see the object data, and it is not the same, for this reason I made the displayName the same as the blogList. To have the name that corresponds.